python3Packages.httpx: 0.19.0 -> 0.21.1

fabaff opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change
Update to latest upstream release 0.20.0

Change log: https://github.com/encode/httpx/releases/tag/0.20.0

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

fabaff committed:
python3Packages.httpx: 0.19.0 -> 0.20.0

fabaff committed:
python3Packages.httpx-socks: 0.4.1 -> 0.5.0

fabaff committed:
python3Packages.datasette: 0.58.1 -> 0.59

fabaff committed:
python3Packages.elmax: 0.1.3 -> 0.1.4

fabaff commented:
Depends on #141684 as `websockets-10.0` is needed.

fabaff commented:
Result of `nixpkgs-review pr 141974` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>32 packages failed to build:</summary>
  <ul>
    <li>dyndnsc</li>
    <li>home-assistant</li>
    <li>pylode</li>
    <li>python38Packages.cirq</li>
    <li>python38Packages.cirq-rigetti</li>
    <li>python38Packages.datasette-template-sql</li>
    <li>python38Packages.entrance</li>
    <li>python38Packages.entrance-with-router-features</li>
    <li>python38Packages.envoy-reader</li>
    <li>python38Packages.hug</li>
    <li>python38Packages.json-logging</li>
    <li>python38Packages.mailman-hyperkitty</li>
    <li>python38Packages.pytest-sanic</li>
    <li>python38Packages.sanic</li>
    <li>python38Packages.sanic-auth</li>
    <li>python38Packages.sanic-testing</li>
    <li>python38Packages.sunpy</li>
    <li>python39Packages.cirq</li>
    <li>python39Packages.cirq-rigetti</li>
    <li>python39Packages.datasette-template-sql</li>
    <li>python39Packages.entrance</li>
    <li>python39Packages.entrance-with-router-features</li>
    <li>python39Packages.envoy-reader</li>
    <li>python39Packages.hug</li>
    <li>python39Packages.json-logging</li>
    <li>python39Packages.pytest-sanic</li>
    <li>python39Packages.sanic</li>
    <li>python39Packages.sanic-auth</li>
    <li>python39Packages.sanic-testing</li>
    <li>python39Packages.sunpy</li>
    <li>theharvester</li>
    <li>wapiti</li>
  </ul>
</details>
<details>
  <summary>89 packages built:</summary>
  <ul>
    <li>apache-airflow (python39Packages.apache-airflow)</li>
    <li>datasette (python39Packages.datasette)</li>
    <li>faraday-cli</li>
    <li>gigalixir (python39Packages.gigalixir)</li>
    <li>lexicon</li>
    <li>mailman</li>
    <li>python38Packages.PyRMVtransport</li>
    <li>python38Packages.amcrest</li>
    <li>python38Packages.apache-airflow</li>
    <li>python38Packages.asyncwhois</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.axis</li>
    <li>python38Packages.datasette</li>
    <li>python38Packages.denonavr</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.elmax</li>
    <li>python38Packages.falcon</li>
    <li>python38Packages.fastapi</li>
    <li>python38Packages.gigalixir</li>
    <li>python38Packages.ha-philipsjs</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.httpx-ntlm</li>
    <li>python38Packages.httpx-socks</li>
    <li>python38Packages.iotawattpy</li>
    <li>python38Packages.ipfshttpclient</li>
    <li>python38Packages.ismartgate</li>
    <li>python38Packages.jupyterlab</li>
    <li>python38Packages.jupyterlab-git</li>
    <li>python38Packages.jupyterlab_server</li>
    <li>python38Packages.jupytext</li>
    <li>python38Packages.mailman (python39Packages.mailman)</li>
    <li>python38Packages.openapi-core</li>
    <li>python38Packages.pylitterbot</li>
    <li>python38Packages.pypoint</li>
    <li>python38Packages.pyquil</li>
    <li>python38Packages.pytest-httpx</li>
    <li>python38Packages.qcs-api-client</li>
    <li>python38Packages.respx</li>
    <li>python38Packages.rollbar</li>
    <li>python38Packages.sanic-routing</li>
    <li>python38Packages.simple-rest-client</li>
    <li>python38Packages.slowapi</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.total-connect-client</li>
    <li>python38Packages.webargs</li>
    <li>python38Packages.whodap</li>
    <li>python38Packages.youtube-search-python</li>
    <li>python38Packages.zeep</li>
    <li>python39Packages.PyRMVtransport</li>
    <li>python39Packages.amcrest</li>
    <li>python39Packages.asyncwhois</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.axis</li>
    <li>python39Packages.denonavr</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.elmax</li>
    <li>python39Packages.falcon</li>
    <li>python39Packages.fastapi</li>
    <li>python39Packages.ha-philipsjs</li>
    <li>python39Packages.httpx</li>
    <li>python39Packages.httpx-ntlm</li>
    <li>python39Packages.httpx-socks</li>
    <li>python39Packages.iotawattpy</li>
    <li>python39Packages.ipfshttpclient</li>
    <li>python39Packages.ismartgate</li>
    <li>python39Packages.jupyterlab</li>
    <li>python39Packages.jupyterlab-git</li>
    <li>python39Packages.jupyterlab_server</li>
    <li>python39Packages.jupytext</li>
    <li>python39Packages.mailman-hyperkitty</li>
    <li>python39Packages.openapi-core</li>
    <li>python39Packages.pylitterbot</li>
    <li>python39Packages.pypoint</li>
    <li>python39Packages.pyquil</li>
    <li>python39Packages.pytest-httpx</li>
    <li>python39Packages.qcs-api-client</li>
    <li>python39Packages.respx</li>
    <li>python39Packages.rollbar</li>
    <li>python39Packages.sanic-routing</li>
    <li>python39Packages.simple-rest-client</li>
    <li>python39Packages.slowapi</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.total-connect-client</li>
    <li>python39Packages.webargs</li>
    <li>python39Packages.whodap</li>
    <li>python39Packages.youtube-search-python</li>
    <li>python39Packages.zeep</li>
    <li>soco-cli</li>
    <li>ytmdl</li>
  </ul>
</details>

SuperSandro2000 commented on pkgs/development/python-modules/prov/default.nix:
```diff
@@ -28,8 +28,16 @@ buildPythonPackage rec {
     pydot
   ];
 
+
+  # Multiple tests are out-dated and failing
```
One new line to much.

fabaff replied:
Thanks

fabaff committed:
python3Packages.httpx-socks: 0.5.0 -> 0.5.1

fabaff committed:
wapiti: 3.0.5 -> 3.0.7

fabaff committed:
python3Packages.envoy-reader: allow later httpx releases

fabaff committed:
python3Packages.sanic-auth: support later httpx releases

fabaff committed:
python3Packages.pytest-httpx: 0.13.0 -> 0.14.0

fabaff commented:
Result of `nixpkgs-review pr 141974` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages marked as broken and skipped:</summary>
  <ul>
    <li>cwltool</li>
    <li>python38Packages.nipype</li>
    <li>python38Packages.pytest-sanic</li>
    <li>python38Packages.rdflib-jsonld</li>
    <li>python38Packages.schema-salad</li>
    <li>python39Packages.nipype</li>
    <li>python39Packages.pytest-sanic</li>
    <li>python39Packages.rdflib-jsonld</li>
    <li>python39Packages.schema-salad</li>
  </ul>
</details>
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.cirq</li>
    <li>python38Packages.cirq-rigetti</li>
    <li>python38Packages.hug</li>
    <li>python38Packages.mailman-hyperkitty</li>
    <li>python39Packages.cirq</li>
    <li>python39Packages.cirq-rigetti</li>
    <li>python39Packages.hug</li>
  </ul>
</details>
<details>
  <summary>142 packages built:</summary>
  <ul>
    <li>apache-airflow (python39Packages.apache-airflow)</li>
    <li>carla</li>
    <li>datasette (python39Packages.datasette)</li>
    <li>dyndnsc</li>
    <li>faraday-cli</li>
    <li>gigalixir (python39Packages.gigalixir)</li>
    <li>github-to-sqlite (python39Packages.github-to-sqlite)</li>
    <li>ingen</li>
    <li>lexicon</li>
    <li>mailman</li>
    <li>odoo</li>
    <li>pantheon-tweaks</li>
    <li>pantheon.elementary-code</li>
    <li>pantheon.elementary-files</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-music</li>
    <li>pantheon.switchboard-plug-security-privacy</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>pylode</li>
    <li>python38Packages.PyRMVtransport</li>
    <li>python38Packages.amcrest</li>
    <li>python38Packages.apache-airflow</li>
    <li>python38Packages.asyncwhois</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.axis</li>
    <li>python38Packages.datasette</li>
    <li>python38Packages.datasette-template-sql</li>
    <li>python38Packages.denonavr</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.elmax</li>
    <li>python38Packages.entrance</li>
    <li>python38Packages.entrance-with-router-features</li>
    <li>python38Packages.envoy-reader</li>
    <li>python38Packages.falcon</li>
    <li>python38Packages.fastapi</li>
    <li>python38Packages.gigalixir</li>
    <li>python38Packages.github-to-sqlite</li>
    <li>python38Packages.ha-philipsjs</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.httpx-ntlm</li>
    <li>python38Packages.httpx-socks</li>
    <li>python38Packages.iotawattpy</li>
    <li>python38Packages.ipfshttpclient</li>
    <li>python38Packages.ismartgate</li>
    <li>python38Packages.json-logging</li>
    <li>python38Packages.jupyterlab</li>
    <li>python38Packages.jupyterlab-git</li>
    <li>python38Packages.jupyterlab_server</li>
    <li>python38Packages.jupytext</li>
    <li>python38Packages.mailman (python39Packages.mailman)</li>
    <li>python38Packages.openapi-core</li>
    <li>python38Packages.parfive</li>
    <li>python38Packages.prov</li>
    <li>python38Packages.pylitterbot</li>
    <li>python38Packages.pypoint</li>
    <li>python38Packages.pyppeteer</li>
    <li>python38Packages.pyquil</li>
    <li>python38Packages.pytest-httpx</li>
    <li>python38Packages.qcs-api-client</li>
    <li>python38Packages.rdflib</li>
    <li>python38Packages.respx</li>
    <li>python38Packages.rollbar</li>
    <li>python38Packages.sanic</li>
    <li>python38Packages.sanic-auth</li>
    <li>python38Packages.sanic-testing</li>
    <li>python38Packages.scancode-toolkit</li>
    <li>python38Packages.simple-rest-client</li>
    <li>python38Packages.slowapi</li>
    <li>python38Packages.spdx-tools</li>
    <li>python38Packages.sqlite-utils</li>
    <li>python38Packages.sunpy</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.total-connect-client</li>
    <li>python38Packages.webargs</li>
    <li>python38Packages.whodap</li>
    <li>python38Packages.youtube-search-python</li>
    <li>python38Packages.zeep</li>
    <li>python38Packages.zeitgeist</li>
    <li>python39Packages.PyRMVtransport</li>
    <li>python39Packages.amcrest</li>
    <li>python39Packages.asyncwhois</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.axis</li>
    <li>python39Packages.datasette-template-sql</li>
    <li>python39Packages.denonavr</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.elmax</li>
    <li>python39Packages.entrance</li>
    <li>python39Packages.entrance-with-router-features</li>
    <li>python39Packages.envoy-reader</li>
    <li>python39Packages.falcon</li>
    <li>python39Packages.fastapi</li>
    <li>python39Packages.ha-philipsjs</li>
    <li>python39Packages.httpx</li>
    <li>python39Packages.httpx-ntlm</li>
    <li>python39Packages.httpx-socks</li>
    <li>python39Packages.iotawattpy</li>
    <li>python39Packages.ipfshttpclient</li>
    <li>python39Packages.ismartgate</li>
    <li>python39Packages.json-logging</li>
    <li>python39Packages.jupyterlab</li>
    <li>python39Packages.jupyterlab-git</li>
    <li>python39Packages.jupyterlab_server</li>
    <li>python39Packages.jupytext</li>
    <li>python39Packages.mailman-hyperkitty</li>
    <li>python39Packages.openapi-core</li>
    <li>python39Packages.parfive</li>
    <li>python39Packages.prov</li>
    <li>python39Packages.pylitterbot</li>
    <li>python39Packages.pypoint</li>
    <li>python39Packages.pyppeteer</li>
    <li>python39Packages.pyquil</li>
    <li>python39Packages.pytest-httpx</li>
    <li>python39Packages.qcs-api-client</li>
    <li>python39Packages.rdflib</li>
    <li>python39Packages.respx</li>
    <li>python39Packages.rollbar</li>
    <li>python39Packages.sanic</li>
    <li>python39Packages.sanic-auth</li>
    <li>python39Packages.sanic-testing</li>
    <li>python39Packages.scancode-toolkit</li>
    <li>python39Packages.simple-rest-client</li>
    <li>python39Packages.slowapi</li>
    <li>python39Packages.spdx-tools</li>
    <li>sqlite-utils (python39Packages.sqlite-utils)</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.total-connect-client</li>
    <li>python39Packages.webargs</li>
    <li>python39Packages.whodap</li>
    <li>python39Packages.youtube-search-python</li>
    <li>python39Packages.zeep</li>
    <li>python39Packages.zeitgeist</li>
    <li>soco-cli</li>
    <li>synapse</li>
    <li>theharvester</li>
    <li>urlwatch</li>
    <li>wapiti</li>
    <li>ytmdl</li>
    <li>zeitgeist</li>
  </ul>
</details>

fabaff committed:
python3Packages.respx: 0.17.1 -> 0.18.2

fabaff committed:
python3Packages.httpx-socks: patch certificate for tests

fabaff committed:
home-assistant: relax httpx constraint

fabaff commented:
Result of `nixpkgs-review pr 141974` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.hug</li>
    <li>python38Packages.pytest-sanic</li>
    <li>python39Packages.hug</li>
    <li>python39Packages.pytest-sanic</li>
  </ul>
</details>
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.datasette-template-sql</li>
    <li>python39Packages.datasette-template-sql</li>
  </ul>
</details>
<details>
  <summary>112 packages built:</summary>
  <ul>
    <li>apache-airflow (python39Packages.apache-airflow)</li>
    <li>datasette (python39Packages.datasette)</li>
    <li>dyndnsc</li>
    <li>faraday-cli</li>
    <li>gigalixir (python39Packages.gigalixir)</li>
    <li>lexicon</li>
    <li>mailman</li>
    <li>odoo</li>
    <li>pylode</li>
    <li>python38Packages.amcrest</li>
    <li>python38Packages.apache-airflow</li>
    <li>python38Packages.asyncwhois</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.axis</li>
    <li>python38Packages.cirq</li>
    <li>python38Packages.cirq-rigetti</li>
    <li>python38Packages.datasette</li>
    <li>python38Packages.denonavr</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.elmax</li>
    <li>python38Packages.entrance</li>
    <li>python38Packages.entrance-with-router-features</li>
    <li>python38Packages.envoy-reader</li>
    <li>python38Packages.falcon</li>
    <li>python38Packages.fastapi</li>
    <li>python38Packages.gigalixir</li>
    <li>python38Packages.ha-philipsjs</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.httpx-ntlm</li>
    <li>python38Packages.httpx-socks</li>
    <li>python38Packages.iotawattpy</li>
    <li>python38Packages.ipfshttpclient</li>
    <li>python38Packages.ismartgate</li>
    <li>python38Packages.json-logging</li>
    <li>python38Packages.jupyterlab</li>
    <li>python38Packages.jupyterlab-git</li>
    <li>python38Packages.jupyterlab_server</li>
    <li>python38Packages.jupytext</li>
    <li>python38Packages.mailman (python39Packages.mailman)</li>
    <li>python38Packages.openapi-core</li>
    <li>python38Packages.pylitterbot</li>
    <li>python38Packages.pypoint</li>
    <li>python38Packages.pyquil</li>
    <li>python38Packages.pyrmvtransport</li>
    <li>python38Packages.pytest-httpx</li>
    <li>python38Packages.qcs-api-client</li>
    <li>python38Packages.respx</li>
    <li>python38Packages.rollbar</li>
    <li>python38Packages.sanic</li>
    <li>python38Packages.sanic-auth</li>
    <li>python38Packages.sanic-testing</li>
    <li>python38Packages.simple-rest-client</li>
    <li>python38Packages.slowapi</li>
    <li>python38Packages.sunpy</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.total-connect-client</li>
    <li>python38Packages.webargs</li>
    <li>python38Packages.whodap</li>
    <li>python38Packages.youtube-search-python</li>
    <li>python38Packages.zeep</li>
    <li>python39Packages.amcrest</li>
    <li>python39Packages.asyncwhois</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.axis</li>
    <li>python39Packages.cirq</li>
    <li>python39Packages.cirq-rigetti</li>
    <li>python39Packages.denonavr</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.elmax</li>
    <li>python39Packages.entrance</li>
    <li>python39Packages.entrance-with-router-features</li>
    <li>python39Packages.envoy-reader</li>
    <li>python39Packages.falcon</li>
    <li>python39Packages.fastapi</li>
    <li>python39Packages.ha-philipsjs</li>
    <li>python39Packages.httpx</li>
    <li>python39Packages.httpx-ntlm</li>
    <li>python39Packages.httpx-socks</li>
    <li>python39Packages.iotawattpy</li>
    <li>python39Packages.ipfshttpclient</li>
    <li>python39Packages.ismartgate</li>
    <li>python39Packages.json-logging</li>
    <li>python39Packages.jupyterlab</li>
    <li>python39Packages.jupyterlab-git</li>
    <li>python39Packages.jupyterlab_server</li>
    <li>python39Packages.jupytext</li>
    <li>python39Packages.mailman-hyperkitty</li>
    <li>python39Packages.openapi-core</li>
    <li>python39Packages.pylitterbot</li>
    <li>python39Packages.pypoint</li>
    <li>python39Packages.pyquil</li>
    <li>python39Packages.pyrmvtransport</li>
    <li>python39Packages.pytest-httpx</li>
    <li>python39Packages.qcs-api-client</li>
    <li>python39Packages.respx</li>
    <li>python39Packages.rollbar</li>
    <li>python39Packages.sanic</li>
    <li>python39Packages.sanic-auth</li>
    <li>python39Packages.sanic-testing</li>
    <li>python39Packages.simple-rest-client</li>
    <li>python39Packages.slowapi</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.total-connect-client</li>
    <li>python39Packages.webargs</li>
    <li>python39Packages.whodap</li>
    <li>python39Packages.youtube-search-python</li>
    <li>python39Packages.zeep</li>
    <li>soco-cli</li>
    <li>theharvester</li>
    <li>wapiti</li>
    <li>ytmdl</li>
  </ul>
</details>

fabaff commented:
- home-assistant: one failure caused by `nam` integration
- python38Packages.datasette-template-sql: will be access with `sqlite-utils` which is in `staging`
- https://hydra.nixos.org/build/158060401 should be fixed with the added patch for `httpx-socks`

ZHF: #144627

mweinelt commented:
Wondering if this is too big before the branch-off.

Result of `nixpkgs-review pr 141974` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.hug</li>
    <li>python38Packages.pytest-sanic</li>
    <li>python39Packages.hug</li>
    <li>python39Packages.pytest-sanic</li>
  </ul>
</details>
<details>
  <summary>10 packages failed to build:</summary>
  <ul>
    <li>dyndnsc</li>
    <li>home-assistant</li>
    <li>python38Packages.datasette-template-sql</li>
    <li>python39Packages.datasette-template-sql</li>
    <li>python39Packages.entrance</li>
    <li>python39Packages.entrance-with-router-features</li>
    <li>python39Packages.json-logging</li>
    <li>python39Packages.sanic</li>
    <li>python39Packages.sanic-auth</li>
    <li>python39Packages.sanic-testing</li>
  </ul>
</details>
<details>
  <summary>105 packages built:</summary>
  <ul>
    <li>apache-airflow (python39Packages.apache-airflow)</li>
    <li>datasette (python39Packages.datasette)</li>
    <li>faraday-cli</li>
    <li>gigalixir (python39Packages.gigalixir)</li>
    <li>lexicon</li>
    <li>mailman</li>
    <li>odoo</li>
    <li>pylode</li>
    <li>python38Packages.amcrest</li>
    <li>python38Packages.apache-airflow</li>
    <li>python38Packages.asyncwhois</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.axis</li>
    <li>python38Packages.cirq</li>
    <li>python38Packages.cirq-rigetti</li>
    <li>python38Packages.datasette</li>
    <li>python38Packages.denonavr</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.elmax</li>
    <li>python38Packages.entrance</li>
    <li>python38Packages.entrance-with-router-features</li>
    <li>python38Packages.envoy-reader</li>
    <li>python38Packages.falcon</li>
    <li>python38Packages.fastapi</li>
    <li>python38Packages.gigalixir</li>
    <li>python38Packages.ha-philipsjs</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.httpx-ntlm</li>
    <li>python38Packages.httpx-socks</li>
    <li>python38Packages.iotawattpy</li>
    <li>python38Packages.ipfshttpclient</li>
    <li>python38Packages.ismartgate</li>
    <li>python38Packages.json-logging</li>
    <li>python38Packages.jupyterlab</li>
    <li>python38Packages.jupyterlab-git</li>
    <li>python38Packages.jupyterlab_server</li>
    <li>python38Packages.jupytext</li>
    <li>python38Packages.mailman (python39Packages.mailman)</li>
    <li>python38Packages.openapi-core</li>
    <li>python38Packages.pylitterbot</li>
    <li>python38Packages.pypoint</li>
    <li>python38Packages.pyquil</li>
    <li>python38Packages.pyrmvtransport</li>
    <li>python38Packages.pytest-httpx</li>
    <li>python38Packages.qcs-api-client</li>
    <li>python38Packages.respx</li>
    <li>python38Packages.rollbar</li>
    <li>python38Packages.sanic</li>
    <li>python38Packages.sanic-auth</li>
    <li>python38Packages.sanic-testing</li>
    <li>python38Packages.simple-rest-client</li>
    <li>python38Packages.slowapi</li>
    <li>python38Packages.sunpy</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.total-connect-client</li>
    <li>python38Packages.webargs</li>
    <li>python38Packages.whodap</li>
    <li>python38Packages.youtube-search-python</li>
    <li>python38Packages.zeep</li>
    <li>python39Packages.amcrest</li>
    <li>python39Packages.asyncwhois</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.axis</li>
    <li>python39Packages.cirq</li>
    <li>python39Packages.cirq-rigetti</li>
    <li>python39Packages.denonavr</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.elmax</li>
    <li>python39Packages.envoy-reader</li>
    <li>python39Packages.falcon</li>
    <li>python39Packages.fastapi</li>
    <li>python39Packages.ha-philipsjs</li>
    <li>python39Packages.httpx</li>
    <li>python39Packages.httpx-ntlm</li>
    <li>python39Packages.httpx-socks</li>
    <li>python39Packages.iotawattpy</li>
    <li>python39Packages.ipfshttpclient</li>
    <li>python39Packages.ismartgate</li>
    <li>python39Packages.jupyterlab</li>
    <li>python39Packages.jupyterlab-git</li>
    <li>python39Packages.jupyterlab_server</li>
    <li>python39Packages.jupytext</li>
    <li>python39Packages.mailman-hyperkitty</li>
    <li>python39Packages.openapi-core</li>
    <li>python39Packages.pylitterbot</li>
    <li>python39Packages.pypoint</li>
    <li>python39Packages.pyquil</li>
    <li>python39Packages.pyrmvtransport</li>
    <li>python39Packages.pytest-httpx</li>
    <li>python39Packages.qcs-api-client</li>
    <li>python39Packages.respx</li>
    <li>python39Packages.rollbar</li>
    <li>python39Packages.simple-rest-client</li>
    <li>python39Packages.slowapi</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.total-connect-client</li>
    <li>python39Packages.webargs</li>
    <li>python39Packages.whodap</li>
    <li>python39Packages.youtube-search-python</li>
    <li>python39Packages.zeep</li>
    <li>soco-cli</li>
    <li>theharvester</li>
    <li>wapiti</li>
    <li>ytmdl</li>
  </ul>
</details>

mweinelt commented:
> home-assistant: one failure caused by `nam` integration

Because `nettigo-air-monitor` was bumped to incompatible release 1.2.0 in #145686, while no pin was provided.

dotlambda committed:
python3Packages.luftdaten: 0.6.5 -> 0.7.0

https://github.com/home-assistant-ecosystem/python-luftdaten/blob/0.7.0/CHANGES.rst

fabaff committed:
python3Packages.httpx: 0.20.0 -> 0.21.1

fabaff committed:
python3Packages.httpx-socks: 0.5.1 -> 0.6.0

fabaff committed:
python3Packages.httpcore: 0.13.7 -> 0.14.3

fabaff committed:
python3Packages.respx: 0.18.2 -> 0.19.0

fabaff committed:
python3Packages.requests-cache: disable flaky tests

fabaff committed:
python3Packages.python-socks: 1.2.4 -> 2.0.0

fabaff committed:
python3Packages.httpx-socks: 0.6.0 -> 0.7.0

fabaff commented:
Result of `nixpkgs-review pr 141974` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>24 packages marked as broken and skipped:</summary>
  <ul>
    <li>awsebcli</li>
    <li>buildbot</li>
    <li>buildbot-full</li>
    <li>buildbot-ui</li>
    <li>mlflow-server</li>
    <li>python38Packages.buildbot</li>
    <li>python38Packages.buildbot-full</li>
    <li>python38Packages.buildbot-ui</li>
    <li>python38Packages.gradient</li>
    <li>python38Packages.hug</li>
    <li>python38Packages.mlflow</li>
    <li>python38Packages.pytest-sanic</li>
    <li>python38Packages.rl-coach</li>
    <li>python38Packages.tvdb_api</li>
    <li>python39Packages.buildbot</li>
    <li>python39Packages.buildbot-full</li>
    <li>python39Packages.buildbot-ui</li>
    <li>python39Packages.gradient</li>
    <li>python39Packages.hug</li>
    <li>python39Packages.mlflow</li>
    <li>python39Packages.pytest-sanic</li>
    <li>python39Packages.rl-coach</li>
    <li>python39Packages.tvdb_api</li>
    <li>tvnamer</li>
  </ul>
</details>
<details>
  <summary>36 packages failed to build:</summary>
  <ul>
    <li>dyndnsc</li>
    <li>electrum</li>
    <li>heisenbridge</li>
    <li>home-assistant</li>
    <li>matrix-commander</li>
    <li>mirage-im</li>
    <li>opsdroid</li>
    <li>pantalaimon</li>
    <li>pantalaimon-headless</li>
    <li>python38Packages.cirq</li>
    <li>python38Packages.cirq-rigetti</li>
    <li>python38Packages.elmax</li>
    <li>python38Packages.entrance</li>
    <li>python38Packages.entrance-with-router-features</li>
    <li>python38Packages.homematicip</li>
    <li>python38Packages.json-logging</li>
    <li>python38Packages.luftdaten</li>
    <li>python38Packages.mailman</li>
    <li>python38Packages.matrix-nio</li>
    <li>python38Packages.sanic</li>
    <li>python38Packages.sanic-auth</li>
    <li>python38Packages.sanic-testing</li>
    <li>python39Packages.cirq</li>
    <li>python39Packages.cirq-rigetti</li>
    <li>python39Packages.elmax</li>
    <li>python39Packages.entrance</li>
    <li>python39Packages.entrance-with-router-features</li>
    <li>python39Packages.homematicip</li>
    <li>python39Packages.json-logging</li>
    <li>python39Packages.luftdaten</li>
    <li>python39Packages.matrix-nio</li>
    <li>python39Packages.sanic</li>
    <li>python39Packages.sanic-auth</li>
    <li>python39Packages.sanic-testing</li>
    <li>tts</li>
    <li>weechatScripts.weechat-matrix</li>
  </ul>
</details>
<details>
  <summary>251 packages built:</summary>
  <ul>
    <li>apache-airflow (python39Packages.apache-airflow)</li>
    <li>appdaemon</li>
    <li>arion</li>
    <li>aws-sam-cli</li>
    <li>azure-cli</li>
    <li>checkov</li>
    <li>cryptop</li>
    <li>datasette (python39Packages.datasette)</li>
    <li>docker-compose (docker_compose)</li>
    <li>errbot</li>
    <li>faraday-cli</li>
    <li>giara</li>
    <li>gigalixir (python39Packages.gigalixir)</li>
    <li>jellyfin-mpv-shim</li>
    <li>ledger_agent (python39Packages.ledger_agent)</li>
    <li>lexicon</li>
    <li>mailman</li>
    <li>mnamer</li>
    <li>mpfshell</li>
    <li>ntfy</li>
    <li>odoo</li>
    <li>onionshare</li>
    <li>onionshare-gui</li>
    <li>parquet-tools</li>
    <li>plex-mpv-shim</li>
    <li>pylode</li>
    <li>python38Packages.aioambient</li>
    <li>python38Packages.aiocurrencylayer</li>
    <li>python38Packages.aiohttp-socks</li>
    <li>python38Packages.amcrest</li>
    <li>python38Packages.apache-airflow</li>
    <li>python38Packages.aria2p</li>
    <li>python38Packages.asyncwhois</li>
    <li>python38Packages.authcaptureproxy</li>
    <li>python38Packages.axis</li>
    <li>python38Packages.celery</li>
    <li>python38Packages.chalice</li>
    <li>python38Packages.datasette</li>
    <li>python38Packages.datasette-template-sql</li>
    <li>python38Packages.denonavr</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.devolo-home-control-api</li>
    <li>python38Packages.devolo-plc-api</li>
    <li>python38Packages.django-raster</li>
    <li>python38Packages.djmail</li>
    <li>python38Packages.docker</li>
    <li>python38Packages.docker-py</li>
    <li>python38Packages.dockerspawner</li>
    <li>python38Packages.envoy-reader</li>
    <li>python38Packages.falcon</li>
    <li>python38Packages.fastapi</li>
    <li>python38Packages.flask-socketio</li>
    <li>python38Packages.flower</li>
    <li>python38Packages.gensim</li>
    <li>python38Packages.gigalixir</li>
    <li>python38Packages.ha-philipsjs</li>
    <li>python38Packages.httpcore</li>
    <li>python38Packages.httpx</li>
    <li>python38Packages.httpx-ntlm</li>
    <li>python38Packages.httpx-socks</li>
    <li>python38Packages.ibm-watson</li>
    <li>python38Packages.iotawattpy</li>
    <li>python38Packages.ipfshttpclient</li>
    <li>python38Packages.ismartgate</li>
    <li>python38Packages.jellyfin-apiclient-python</li>
    <li>python38Packages.jupyter-repo2docker</li>
    <li>python38Packages.jupyter-server-mathjax</li>
    <li>python38Packages.jupyter_server</li>
    <li>python38Packages.jupyterlab</li>
    <li>python38Packages.jupyterlab-git</li>
    <li>python38Packages.jupyterlab_server</li>
    <li>python38Packages.jupytext</li>
    <li>python38Packages.kubernetes</li>
    <li>python38Packages.ledger_agent</li>
    <li>python38Packages.ledgerblue</li>
    <li>python38Packages.libsoundtouch</li>
    <li>python38Packages.moto</li>
    <li>python38Packages.nbclassic</li>
    <li>python38Packages.nbdime</li>
    <li>python38Packages.openapi-core</li>
    <li>python38Packages.openshift</li>
    <li>python38Packages.pathy</li>
    <li>python38Packages.plexapi</li>
    <li>python38Packages.praw</li>
    <li>python38Packages.pushbullet</li>
    <li>python38Packages.pyfireservicerota</li>
    <li>python38Packages.pylitterbot</li>
    <li>python38Packages.pypoint</li>
    <li>python38Packages.pyquil</li>
    <li>python38Packages.pyrmvtransport</li>
    <li>python38Packages.pysychonaut</li>
    <li>python38Packages.pytest-httpx</li>
    <li>python38Packages.python-engineio</li>
    <li>python38Packages.python-mpv-jsonipc</li>
    <li>python38Packages.python-socketio</li>
    <li>python38Packages.python-socks</li>
    <li>python38Packages.qcs-api-client</li>
    <li>python38Packages.qiskit</li>
    <li>python38Packages.qiskit-ibmq-provider</li>
    <li>python38Packages.requests-cache</li>
    <li>python38Packages.respx</li>
    <li>python38Packages.rollbar</li>
    <li>python38Packages.roonapi</li>
    <li>python38Packages.runway-python</li>
    <li>python38Packages.samsungctl</li>
    <li>python38Packages.samsungtvws</li>
    <li>python38Packages.sense-energy</li>
    <li>python38Packages.simple-rest-client</li>
    <li>python38Packages.slack-sdk</li>
    <li>python38Packages.slackclient</li>
    <li>python38Packages.slowapi</li>
    <li>python38Packages.smart-open (python38Packages.smart_open)</li>
    <li>python38Packages.sopel</li>
    <li>python38Packages.spacy</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.sunpy</li>
    <li>python38Packages.tensorboardx</li>
    <li>python38Packages.tern</li>
    <li>python38Packages.teslajsonpy</li>
    <li>python38Packages.textacy</li>
    <li>python38Packages.total-connect-client</li>
    <li>python38Packages.tuya-iot-py-sdk</li>
    <li>python38Packages.waterfurnace</li>
    <li>python38Packages.webargs</li>
    <li>python38Packages.websocket-client (python38Packages.websocket_client)</li>
    <li>python38Packages.whodap</li>
    <li>python38Packages.youtube-search-python</li>
    <li>python38Packages.zeep</li>
    <li>python39Packages.aioambient</li>
    <li>python39Packages.aiocurrencylayer</li>
    <li>python39Packages.aiohttp-socks</li>
    <li>python39Packages.amcrest</li>
    <li>python39Packages.aria2p</li>
    <li>python39Packages.asyncwhois</li>
    <li>python39Packages.authcaptureproxy</li>
    <li>python39Packages.axis</li>
    <li>python39Packages.celery</li>
    <li>python39Packages.chalice</li>
    <li>python39Packages.datasette-template-sql</li>
    <li>python39Packages.denonavr</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.devolo-home-control-api</li>
    <li>python39Packages.devolo-plc-api</li>
    <li>python39Packages.django-raster</li>
    <li>python39Packages.djmail</li>
    <li>python39Packages.docker</li>
    <li>python39Packages.docker-py</li>
    <li>python39Packages.dockerspawner</li>
    <li>python39Packages.envoy-reader</li>
    <li>python39Packages.falcon</li>
    <li>python39Packages.fastapi</li>
    <li>python39Packages.flask-socketio</li>
    <li>python39Packages.flower</li>
    <li>python39Packages.gensim</li>
    <li>python39Packages.ha-philipsjs</li>
    <li>python39Packages.httpcore</li>
    <li>python39Packages.httpx</li>
    <li>python39Packages.httpx-ntlm</li>
    <li>python39Packages.httpx-socks</li>
    <li>python39Packages.ibm-watson</li>
    <li>python39Packages.iotawattpy</li>
    <li>python39Packages.ipfshttpclient</li>
    <li>python39Packages.ismartgate</li>
    <li>python39Packages.jellyfin-apiclient-python</li>
    <li>python39Packages.jupyter-repo2docker</li>
    <li>python39Packages.jupyter-server-mathjax</li>
    <li>python39Packages.jupyter_server</li>
    <li>python39Packages.jupyterlab</li>
    <li>python39Packages.jupyterlab-git</li>
    <li>python39Packages.jupyterlab_server</li>
    <li>python39Packages.jupytext</li>
    <li>python39Packages.kubernetes</li>
    <li>python39Packages.ledgerblue</li>
    <li>python39Packages.libsoundtouch</li>
    <li>python39Packages.mailman</li>
    <li>python39Packages.mailman-hyperkitty</li>
    <li>python39Packages.moto</li>
    <li>python39Packages.nbclassic</li>
    <li>python39Packages.nbdime</li>
    <li>python39Packages.openapi-core</li>
    <li>python39Packages.openshift</li>
    <li>python39Packages.pathy</li>
    <li>python39Packages.plexapi</li>
    <li>python39Packages.praw</li>
    <li>python39Packages.pushbullet</li>
    <li>python39Packages.pyfireservicerota</li>
    <li>python39Packages.pylitterbot</li>
    <li>python39Packages.pypoint</li>
    <li>python39Packages.pyquil</li>
    <li>python39Packages.pyrmvtransport</li>
    <li>python39Packages.pysychonaut</li>
    <li>python39Packages.pytest-httpx</li>
    <li>python39Packages.python-engineio</li>
    <li>python39Packages.python-mpv-jsonipc</li>
    <li>python39Packages.python-socketio</li>
    <li>python39Packages.python-socks</li>
    <li>python39Packages.qcs-api-client</li>
    <li>python39Packages.qiskit</li>
    <li>python39Packages.qiskit-ibmq-provider</li>
    <li>python39Packages.requests-cache</li>
    <li>python39Packages.respx</li>
    <li>python39Packages.rollbar</li>
    <li>python39Packages.roonapi</li>
    <li>python39Packages.runway-python</li>
    <li>python39Packages.samsungctl</li>
    <li>python39Packages.samsungtvws</li>
    <li>python39Packages.sense-energy</li>
    <li>python39Packages.simple-rest-client</li>
    <li>python39Packages.slack-sdk</li>
    <li>python39Packages.slackclient</li>
    <li>python39Packages.slowapi</li>
    <li>python39Packages.smart-open (python39Packages.smart_open)</li>
    <li>python39Packages.sopel</li>
    <li>python39Packages.spacy</li>
    <li>python39Packages.spacy-transformers</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.tensorboardx</li>
    <li>python39Packages.tern</li>
    <li>python39Packages.teslajsonpy</li>
    <li>python39Packages.textacy</li>
    <li>python39Packages.total-connect-client</li>
    <li>python39Packages.tuya-iot-py-sdk</li>
    <li>python39Packages.waterfurnace</li>
    <li>python39Packages.webargs</li>
    <li>python39Packages.websocket-client (python39Packages.websocket_client)</li>
    <li>python39Packages.whodap</li>
    <li>python39Packages.youtube-search-python</li>
    <li>python39Packages.zeep</li>
    <li>snakemake</li>
    <li>soco-cli</li>
    <li>sourcehut.buildsrht</li>
    <li>sourcehut.coresrht</li>
    <li>sourcehut.dispatchsrht</li>
    <li>sourcehut.gitsrht</li>
    <li>sourcehut.hgsrht</li>
    <li>sourcehut.hubsrht</li>
    <li>sourcehut.listssrht</li>
    <li>sourcehut.mansrht</li>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.pastesrht</li>
    <li>sourcehut.todosrht</li>
    <li>streamlink</li>
    <li>streamlink-twitch-gui-bin</li>
    <li>theharvester</li>
    <li>toil</li>
    <li>ulauncher</li>
    <li>urlwatch</li>
    <li>wapiti</li>
    <li>weechatScripts.wee-slack</li>
    <li>ytmdl</li>
    <li>zeronet</li>
  </ul>
</details>

fabaff commented:
Looks like that a couple of commits could be integrated in #147426

fabaff commented:
No longer needed as #147426 has the `httpx` update.
